### ISSUE TITLE
TokenLogger Version 1 

### ISSUE THREAD

**Issue by WorldImpex** [ChildMindInstitute/mindlogger-app#923]
As a user I want to record my behaviors that are associated with points so that am implored to make better decisions 

By using this protocol (https://raw.githubusercontent.com/ReproNim/reproschema/master/protocols/TokenLogger/TokenLogger_schema), we need to:

- visually show the user what each item's value is 
- show an appropriate graph of the user's historical data for review

**Comment by binarybottle** [comment]
Yeah, we need to have an option to present frequency values on the Data tab.

**Comment by WorldImpex** [comment]
Here is the example bar chart that we would want on the dashboard

**Comment by WorldImpex** [comment]
Possibly display green vs red up and down arrows to show the change in points (needs to be visually distinct for color blind as well)
